### DISCUSSION TITLE
After packaging the game into an AAR file and integrating it into the Android application, it doesn't run.

### DISCUSSION THREAD

**Discussion by zhanjunjie2019** [hajimehoshi/ebiten#3019] [Q&A] [answered]
After packaging the game into an AAR file and integrating it into the Android application, it doesn't run. The reason might be that the configuration file cannot be read during initialization. How can I make the program read and write files in relative paths under the Android project?

**Comment by hajimehoshi** [comment, marked as answer]
I've not used filesystem in Android. What about using SQLite or other local storages? https://developer.android.com/training/data-storage/sqlite
